src/hyperlight_wasm/src/sandbox/sandbox_builder.rs

```diff
@@ -25,6 +25,7 @@ use super::proto_wasm_sandbox::ProtoWasmSandbox;
 pub const MIN_STACK_SIZE: u64 = 64 * 1024;
 pub const MIN_INPUT_DATA_SIZE: usize = 192 * 1024;
 pub const MIN_HEAP_SIZE: u64 = 1024 * 1024;
+pub const MIN_FUNCTION_DEFINITION_SIZE: usize = hyperlight_host::func::MIN_HOST_FUNCTION_DEFINITION_SIZE;
```

Comment:
Don't see where this is used or needed?

Reply:
It's imposed by hyperlight_host's `set_host_function_definition_size`.

Reply:
Good point. And all the other consts are not publicly exposed anyway.